packages/cli/src/commands/daemon/start.ts

```diff
@@ -32,6 +32,11 @@ export default class Start extends Command {
       default: 26656,
       required: true,
     }),
+    'rpc-port': flags.integer({
```

Comment:
i recommend to include `tendermint` in the name. We already have multiple RPC servers, engine and tendermint, and we may have also one for cosmos modules.

Reply:
maybe `port` could be renamed to `rpc-port`. and then tendermint's port to `tendermint-rpc-port`, etc..

Reply:
Or we could just remove all notion of rpc (that most people might not know) and just have `port` and `tm-port` or `tendermint-port`

Reply:
good idea, i like `tendermint-port`!

Reply:
Could you also add the prometheus port with a similar name?